app/columns.tsx

```diff
@@ -62,7 +64,7 @@ export type RecipeInfo = {
   nimbusExperiment: NimbusExperiment
   isBranch?: boolean
   branches: BranchInfo[]
```

Comment:
Please add something like 
```
// XXX rename this to branchInfos to avoid confusion with the thing called branches inside nimbus recipes themselves
```